.github/workflows/publish.yml

```diff
@@ -70,3 +70,12 @@ jobs:
           push: true
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
+
+      # Update Readme on Docker Hub
+      - name: Docker Hub Description
```

Comment:
```suggestion
      - name: Publish README to Docker Hub
```

Reply:
@pascalberger Updated the RP with your feedback.